web/js/signup_comps/Auth.js

```diff
@@ -68,16 +83,100 @@ var SocialAuth = React.createClass({
     }
 });
 
+var wasClicked = {
+  "Google": null,
+  "Facebook": null,
+  "Twitter": null,
+  "Tumblr": null,
+  "Reddit": null,
+  "Instagram": null,
+  "Spotify": null
+};
+var sharePoints = 0;
+var clickBtns = 0;
+
 var Auth = React.createClass({
     getInitialState: function () {
         return {
-            data: []
+            fullPoints: false,
+            doneClick: false,
+            checker: null,
+            shitDates: null
         };
     },
 
+    componentDidUpdate: function () {
+      $.fn.fullpage.reBuild();
```

Comment:
fixed

On Mon, Feb 15, 2016 at 11:22 PM, Micha Gorelick <notifications@github.com>
wrote:

> In web/js/signup_comps/Auth.js
> <https://github.com/QuantifiedSelfless/quantifiedSelf/pull/47#discussion_r52972326>
> :
>
> >          };
> >      },
> >
> > +    componentDidUpdate: function () {
> > +      $.fn.fullpage.reBuild();
>
> Do we need this anymore since we don't use fullpage?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/QuantifiedSelfless/quantifiedSelf/pull/47/files#r52972326>
> .
>